### DISCUSSION TITLE
How to solve the problem Copilot Chat say:'Error on conversation request. Check the log for more details.'

### DISCUSSION THREAD

**Discussion by LinKPrian** [community/community#68902] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I tried it 3 days ago and its still worked,but this day I was trying,and it said Error on conversation request. Check the log for more details.copilot still works,but chat doesn't work.
![image](https://github.com/community/community/assets/28661201/9e5ddf8f-3b26-40c4-896a-08b799a64245)

**Comment by harxh-v** [comment]
@LinKPrian 

If you are experiencing an error with Copilot Chat and it was working fine a few days ago, there could be several reasons behind the issue. Here are some steps you can take to try and resolve the problem:

1. **Check for Service Status**: First, check the status of the Copilot. Sometimes, these services may experience outages or technical issues. 

2. **Clear Browser Cache**: If you're using Copilot Chat through a web browser, try clearing your browser's cache and cookies. Sometimes, cached data can cause issues. After clearing the cache, restart your browser and try accessing Copilot Chat again.

3. **Try a Different Browser**: If the issue persists, try accessing Copilot Chat using a different web browser to see if the problem is browser-specific.

4. **Disable Browser Extensions**: Browser extensions or add-ons can sometimes interfere with web applications. Try disabling any browser extensions or add-ons one by one to see if any of them are causing the issue.

5. **Use Incognito/Private Mode**: Try accessing Copilot Chat in incognito or private browsing mode. This mode disables browser extensions and cached data, which can help determine if the issue is related to your browser settings.

If the problem still persists after trying these steps, the issue may be on the service provider's end, and you'll need to wait for them to resolve it. It's also possible that there are temporary server or infrastructure issues affecting the service. In such cases, service providers typically work to resolve these issues as quickly as possible.

**Reply by LinKPrian** [reply]
browser?I used it on vs code insiders

**Comment by jeffseiffert** [comment]
same issue for me

**Comment by jeffseiffert** [comment]
I'm not sure if this will fix it for anyone else but this article fixed it for me. Essentially install win-ca extension. Go to preferences -> settings -> extensions -> win-ca. Change to append mode. Restart VSCode.
https://stackoverflow.com/questions/71367058/self-signed-certificate-in-certificate-chain-on-github-copilot

**Comment by quduoduo** [comment]
back to 1.8.2 will be OK
